lib/oregon_digital/derivatives/processors/ocr_processor.rb

```diff
@@ -0,0 +1,25 @@
+module OregonDigital::Derivatives::Processors
+  class OcrProcessor < Processor
+    attr_accessor :file, :path
+    def initialize(file, options={})
+      @file = file
+      @path = options.fetch(:path)
+    end
+
+    def run
+      temporary_file do |f|
+        create_path
+        result = `pdftotext -enc UTF-8 '#{f.path}' '#{path}' -bbox 2>&1`
```

Comment:
Not sure if we care, but for oregon news we've slightly altered pdftotext to give us more details: https://github.com/jechols/pdftotext (adds some layout pieces so text flow can be understood more easily)